From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: diaaney <av6cts@gmail.com>
Date: Mon, 13 Apr 2026 10:08:18 -0700
Subject: [PATCH] rapier - 1.8 combat: disable mace smash attack and fall
 damage bonus


diff --git a/net/minecraft/world/item/MaceItem.java b/net/minecraft/world/item/MaceItem.java
index c9e6cf2643333a49cda78bbe2956b0f8a9b8a996..d18118bcebad7976370aeeac675de4459f4bdb6f 100644
--- a/net/minecraft/world/item/MaceItem.java
+++ b/net/minecraft/world/item/MaceItem.java
@@ -166,7 +166,9 @@ public class MaceItem extends Item {
     }
 
     public static boolean canSmashAttack(LivingEntity entity) {
-        return entity.fallDistance > 1.5 && !entity.isFallFlying();
+        // rapier start - 1.8 combat: disable mace smash attack (no fall damage bonus, no aoe knockback)
+        return false;
+        // rapier end
     }
 
     @Override
